src/components/MyKiva/MyKivaCard.vue

```diff
@@ -5,20 +5,30 @@
 		:style="{ backgroundImage: `url(${bgImage})` }"
 	>
 		<div
-			v-if="tagText"
-			class="tw-absolute tw-bg-secondary tw-rounded tw-px-1.5 tw-py-0.5 tw-text-small tw-left-1.5 tw-top-1.5
-				tw-drop-shadow-sm tw-font-medium"
+			v-if="images.length"
```

Comment:
What does the carousel look like with a single image?

Reply:
Nice call!
Difficult with same image size and controls dots, we can make it larger for single image case so.
But at least from design, same height as, for multiple case, having dots there just one row of text, and for single image case two rows of text. They are not intentionally to be shown together.

<img width="339" height="354" alt="Screenshot 2025-07-23 at 11 39 51 a m" src="https://github.com/user-attachments/assets/2ab723c6-b7c8-4a75-b36c-37305c27706b" />
<img width="338" height="347" alt="Screenshot 2025-07-23 at 11 39 57 a m" src="https://github.com/user-attachments/assets/c3b2307f-66f1-4db0-96d4-158a29251da2" />

Reply:
Ok I just figured this could end up in a carousel alongside other cards.

I think you could use flex and grid ensure the heights are the same across all cards like we did in story cards. CTA/Buttons remain pinned to the bottom.

Reply:
Oh yeah, in that case we can fix it with a parent wrapper, not needed for now. Thanks!